src/main/java/de/komoot/photon/query/PhotonRequestFactory.java

```diff
@@ -17,12 +18,22 @@
 public class PhotonRequestFactory {
     private final LanguageChecker languageChecker;
     private final static GeometryFactory geometryFactory = new GeometryFactory(new PrecisionModel(), 4326);
+    
+    protected static HashSet<String> m_hsRequestQueryParams = new HashSet<>(Arrays.asList("lang", "q", "lon", "lat", "limit", "distanceSort", "osm_tag"));
```

Comment:
it has to be consistent IMO with the other params like osm_tag. So either always camelCase or always under_score IMO :) ... see https://github.com/graphhopper/graphhopper/issues/682 :)